src/web/components/Button/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+
+export default function Button({
+  children,
+  onClick,
+  color,
+  disabled,
+  size,
+  outlined,
+  inverted,
+  rounded,
+  loading,
+  icon,
+}) {
+  const classes = `button${color && ` is-${color}`}${size && ` is-${size}`}${
+    outlined ? ' is-outlined' : ''
+  }${inverted ? ' is-inverted' : ''}${rounded ? ' is-rounded' : ''}${loading ? ' is-loading' : ''}`;
+  return (
+    <button className={classes} disabled={disabled} onClick={onClick}>
+      {icon && <Icon name={icon} />}
```

Comment:
Why not to use Icon as a children of `Button` instead of creating a tight couple between `Button` and `Icon` ?

If Icon would have been mandatory, it would have been a good idea, but since it's not, I'm not sure we need this coupling